### DISCUSSION TITLE
colspan/rowspan support with table?

### DISCUSSION THREAD

**Discussion by calidion** [marp-team/marp#187] [Q&A] [answered]
It seems html no longer supported.
don't know how to create a table with rowspan/colspan.

**Comment by yhatt** [comment, marked as answer]
> It seems html no longer supported.

_That's not true._ Marp can render HTML since 3 years ago. (marp-team/marp-cli#7)

Enable HTML rendering option by `--html` argument in CLI, or `Markdown: Marp > Enable HTML` extension setting in VS Code extension: https://github.com/marp-team/marp-vscode#enable-html-in-marp-markdown-%EF%B8%8F.

Then, you can use `colspan` and `rowspan` attribute in `<td>` and `<th>` tag in HTML table.

![table](https://user-images.githubusercontent.com/3993388/137893920-a6b23f5a-bd73-435e-bbc5-75482565e212.png)

**Reply by calidion** [reply]
It is great to have this option.

It would perfect to have table col/row span support like [this](https://github.com/shd101wyy/markdown-preview-enhanced/issues/479) .

**Reply by yhatt** [reply]
Marp CLI, for power users, [can extend Markdown syntaxes through JavaScript](https://github.com/marp-team/marp-cli#functional-engine). Probably [markdown-it-multimd-table](https://www.npmjs.com/package/markdown-it-multimd-table) may be helpful for you.

```js
// engine.js
module.exports = ({ marp }) => marp.use(require('markdown-it-multimd-table'))
```

```bash
npm install @marp-team/marp-cli markdown-it-multimd-table --save
npx marp --engine ./engine.js table.md
```

**Reply by calidion** [reply]
@yhatt 

I am currently writing my ppt using the vscode plugin marp.

It would be very nice to have this feature work on the vscode plugin too.

Thanks.
